### ISSUE TITLE
Editor not appear when adding comment

### ISSUE THREAD

**Issue by haleevra** [taqueci/redmine_wysiwyg_editor#30]
visual editor not appear when add comment the standart textile editor is hidden also

**Comment by taqueci** [comment]
Hi @haleevra 

Please tell me:
* Steps to reproduce
* Redmine version
* Installed plugins
* Plugin versions
* Browser version

**Comment by haleevra** [comment]
Environment:
  Redmine version                3.4.6.stable.17048
  Ruby version                   2.3.8-p459 (2018-10-18) [x86_64-linux]
  Rails version                  4.2.8
  Environment                    production
  Database adapter               SQLServer

visual editor appears only after changing issue 
![redmine wysiwig](https://user-images.githubusercontent.com/24476905/47668465-916be480-dbb9-11e8-8938-4acde2e0198f.gif)
status

**Comment by taqueci** [comment]
I guess the issue is caused by plugin.

Please tell me:
* Installed plugins
* Plugin versions
* Browser version

**Comment by haleevra** [comment]
Redmine plugins:
  a_common_libs                  2.4.9
  custom_menu                    1.8.5
  easy_gantt                     1.11
  easy_instant_messages          2016
  extra_queries                  2.3.5
  global_roles                   2.2.3
  mega_calendar                  1.7.0
  redmine_checklists             3.1.11
  redmine_contacts               4.2.1
  redmine_editauthor             0.11.0
  redmine_inline_auto_complete   0.1.0
  redmine_issue_attachments      1.0.4
  redmine_issue_tabs             1.3.1
  redmine_lightbox2              0.4.3
  redmine_people                 1.4.0
  redmine_preview_office         1.0.3
  redmine_project_alias_2        1.0.3
  redmine_wysiwyg_editor         0.2.10
  redmineup_tags                 2.0.4
  rm_user_mentions               1.0.3
  unread_issues                  2.2.2
  usability                      2.3.8

browser - edge

**Comment by haleevra** [comment]
hi, do you have any news on issue?

**Comment by taqueci** [comment]
Hi @haleevra 

I can't create the same environment as yours because:
* I can not get some commercial plugins.
* Some plugins does not have correct baseline tags.

I'm afraid I can not help you.
Please evaluate plugins one by one.

**Comment by TrumanRu** [comment]
Same problem/
My plugins list. I've marked 9 plugins coincident to @haleevra's environment.

plugin | version | presented too
-- | -- | --
**a_common_libs** | 2.5.0 | X
**custom_menu** | 1.8.5 | X
easy_mindmup | 1.0
easy_wbs | 1.5
**extra_queries** | 2.3.5 | X
**global_roles** | 2.2.3 | X
redmine_attribute_quickies | 1.0.2
redmine_base_deface | 0.0.1
redmine_better_overview | 1.0.0
**redmine_checklists** | 3.1.11 | X
redmine_custom_workflows | 0.1.5
redmine_default_custom_query | 1.3.0
redmine_edit_custom_fields | 0.0.5
redmine_editor_preview_tab | 0.1.5
redmine_gemavatar | 1.1
redmine_git_mirror | 0.8.0
**redmine_inline_auto_complete** | 0.1.0 | X
**redmine_issue_tabs** | 1.3.1 | X
redmine_merge_request_links | 2.1.1
redmine_more_code | 1.0.0
redmine_more_context | 1.0.0
redmine_my_page | 0.1.13
redmine_workflow_enhancements | 0.5.0
redmine_xlsx_format_issue_exporter | 0.1.6
scrum | 0.18.0
subtask_columns | 0.0.4
that_email_log | 0.0.1
**unread_issues** | 2.2.2 | X
**usability** | 2.3.8 | X
view_customize | 1.2.2

```
Environment:
  Redmine version                3.4.6.stable
  Ruby version                   2.3.7-p456 (2018-03-28) [x86_64-linux]
  Rails version                  4.2.8
  Environment                    production
  Database adapter               Mysql2
```

**Comment by taqueci** [comment]
The problem is not reproduced in the environment:
```
Environment:
  Redmine version                3.4.6.stable
  Ruby version                   2.3.3-p222 (2016-11-21) [x86_64-linux]
  Rails version                  4.2.8
  Environment                    development
  Database adapter               SQLite
Redmine plugins:
  a_common_libs                  2.5.3
  custom_menu                    1.8.6
  extra_queries                  2.3.7
  global_roles                   2.2.4
  redmine_checklists             3.1.14
  redmine_inline_auto_complete   0.1.0
  redmine_issue_tabs             1.3.2
  redmine_wysiwyg_editor         0.3.4
  unread_issues                  2.2.3
  usability                      2.3.8
```

**Comment by auc-mma** [comment]
I've a similar problem:
If I create a new task ...save it and add a comment...everything is ok
If I crate a task add an description ..save it and add a comment....--> no tap for "visible editor" is displayed

It has nothing to do with oder plugins...you can reprodice it with a fresh redmine installation

**Comment by auc-mma** [comment]
works... now with Version 0.21.1 ! thank you

**Comment by taqueci** [comment]
@haleevra  andd @TrumanRu 

Is the issue resolved by using 0.21.1?

**Comment by taqueci** [comment]
Closed.
Please reopen if the issue can be reproduced.
